hack/update-codegen.sh

```diff
@@ -42,10 +42,22 @@ kube::codegen::gen_openapi \
   --update-report \
   "${KUEUE_ROOT}/apis/visibility"
 
+externals=(
+  "k8s.io/api/core/v1.PodTemplateSpec:k8s.io/client-go/applyconfigurations/core/v1"
+  "k8s.io/api/core/v1.Taint:k8s.io/client-go/applyconfigurations/core/v1"
+  "k8s.io/api/core/v1.Toleration:k8s.io/client-go/applyconfigurations/core/v1"
+)
```

Comment:
What about `resource.Quantity` (https://github.com/kubernetes-sigs/kueue/blob/main/apis/kueue/v1beta1/clusterqueue_types.go#L233) and `corev1.ResourceList` (https://github.com/kubernetes-sigs/kueue/blob/main/apis/kueue/v1beta1/workload_types.go#L147)?

Reply:
I've checked before and it doesn't seem like apply configurations are generated for these types.
Let me double check.

Reply:
Thanks

Reply:
Yes, using this https://github.com/kubernetes/client-go/blob/07d61b66a66d99fee652bb4183c2653ccf30c815/applyconfigurations/core/v1/resourcerequirements.go#L28C19-L29 as a reference, it seems these types are supposed to be used directly in apply configurations.

Reply:
I see, thank you for confirming that!

Reply:
/lgtm
/approve